data/express-router.js

```diff
@@ -0,0 +1,204 @@
+const express = require("express");
+
+const postDb = require("./helpers/postDb.js");
+const userDb = require("./helpers/userDb.js");
+
+function checkUpperCase() {
+  return function(req, res, next) {
+    let userName = req.headers.name;
+    if (userName.toUpperCase === userName) {
+      next();
+    } else {
+      res.status(404).json("Invalid username");
+    }
+  };
+}
+
+const router = express.Router();
+
+// ********** CREATE METHODS ********** //
+
+// POST request to /api/users
+router.post("/users", checkUpperCase, async (req, res) => {
+  try {
+    const user = await userDb.insert(req.body);
+
```

Comment:
Destructuring req.body before you pass it to the database is better practice for security reasons.